From a914b18f67342657cecb3ad347cc028dc2c833e5 Mon Sep 17 00:00:00 2001
From: weideng <wei.a.deng@intel.com>
Date: Fri, 21 Apr 2017 00:52:03 +0000
Subject: [PATCH 0051/1240] Change Trusty Kconfig to build for X86 Arch only

Currently Trusty only works on x86, so the module should never build
for other archs except x86. Add this patch to add 'depends' part on
drivers/trusty/Kconfig to disable them.

Change-Id: Ic18f351696a9c1c31d57621a4af3e8993cc73de5
Signed-off-by: weideng <wei.a.deng@intel.com>
---
 drivers/trusty/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/trusty/Kconfig b/drivers/trusty/Kconfig
index 7b58db5..7d26922 100644
--- a/drivers/trusty/Kconfig
+++ b/drivers/trusty/Kconfig
@@ -6,6 +6,7 @@ menu "Trusty"
 
 config TRUSTY
 	tristate "Trusty"
+	depends on X86
 	default n
 
 config TRUSTY_FIQ
-- 
2.7.4

